client/python/lib/test/config.py

```diff
@@ -30,3 +30,232 @@
     ("model_name", -1, ValueError, f'model_version should be in range <0, {2**63-1}>'),
     ("model_name", 9223372036854775809, ValueError, f'model_version should be in range <0, {2**63-1}>'),
 ]
+
+ADDRESS_VALID = [
+    ("localhost"),
+    ("19.117.63.126"),
+    ("cluster.cloud.iotg.intel.com")
+]
+
+ADDRESS_INVALID = [
+    (19_117_63_126, TypeError, "address type should be string, but is int"),
+    (['localhost'], TypeError, "address type should be string, but is list"),
+    ("127.14", ValueError, "address is not valid"),
+    ("intel.-com", ValueError, "address is not valid")
+]
+
+PORT_VALID = [
+    (9000),
+    (2**16-1),
+]
+
+PORT_INVALID = [
+    ("9000", TypeError, "port type should be int, but is type str"),
+    ([2**16-1], TypeError, "port type should be int, but is type list"),
+    (2**16, ValueError, f"port should be in range <0, {2**16-1}>"),
+    (-1, ValueError, f"port should be in range <0, {2**16-1}>")
+]
+
+PATH_VALID = "valid_path"
+
+TLS_CONFIG_VALID = [
+    ({"server_cert_path" : PATH_VALID}, 1),
+    ({"client_key_path" : PATH_VALID, "client_cert_path" : PATH_VALID, "server_cert_path" : PATH_VALID}, 3),
```

Comment:
What are the numbers at the end? Can we have them in consts if they mean something?